.eslintrc

```diff
@@ -120,6 +120,12 @@
       }
     },
     // Package overrides
+    {
+      "files": ["packages/admin-panel/**"],
+      "rules": {
+        "import/no-extraneous-dependencies": "off"
```

Comment:
Was there a particular import that was causing the need to disable this rule?

Reply:
Good question. It was for the dev dependencies which are imported in rollup. However when I thought about it a bit more I realised it was only one file so I have just disabled that rule in the file instead which is much more targeted.